sequencer/broadcast/server.go

```diff
@@ -119,7 +115,7 @@ func (s *Server) genericGetBatch(ctx context.Context, batch *state.Batch) (*pb.G
 		RollupExitRoot:    rollupExitRoot,
 		Timestamp:         uint64(batch.Timestamp.Unix()),
 		Transactions:      transactions,
-		ForcedBatchNumber: forcedBatchNum,
+		ForcedBatchNumber: fb,
```

Comment:
maybe this should b a pointer, as I guess that the first `ForcedBatchNumber` will be 0

Reply:
No, In this case the first forcedBatch will be one